src/systems.rs

```diff
@@ -240,7 +240,7 @@ pub fn process_input_system(
         }
     }
 
-    if !ctrl && !win {
+    if !mac_cmd {
```

Comment:
in theory I agree, but the problem is that ctrl is true when using altrgr (maybe that's the actual root issue 🤔) I guess it wouldn't work then.

Reply:
https://en.wikipedia.org/wiki/AltGr_key#Ctrl+Alt

Ah well, I guess that's why. AltGr implies `ctrl+alt`

Reply:
Then that could be discarded? It's `command` only if it's `ctrl`, not `ctrl+alt`?

Reply:
> only if it's `ctrl`, not `ctrl+alt`

how about `!command || ctrl && alt && cfg!(target_os = "windows")`?

Reply:
According to my manual testing, your suggested change works fine for windows and mac :tada:

I might be able to test the behaviour on a spare raspberry pi but not sure when.

Reply:
I meant that maybe `alt` can be included in the `command` variable computation?

Then here just leave `!command`. I mean this because what happens in windows if you do `altgr+c`/`altgr+v`? Does it work as as copy/paste since `altgr` maps to `ctrl+alt` (if I understood correctly).

Reply:
Following the changes in #156, I mean to integrate it like this:

```rs
let command = if cfg!(target_os = "macos") {
    mac_cmd
} else {
    // Only when `ctrl` is pressed; this skips `altgr` which maps to `ctrl+alt`
    ctrl && !alt
};
```

And then just doing `!control` should work, no? It'll ignore shortcuts, and in macOS chars like `ctrl+a` will skip because the inner `!event.char.is_control()` check. But, in macOS this will allow `ctrl+_` usages that actually produce some character, like: `ctrl+´ => "`.

Reply:
@doup I'd prefer the `altgr` behaviour to be limited just to Windows, as I haven't found any proof that it works like that on any other platform. I haven't invested enough time to understand how `ctrl+_` works in MacOS, but on my ANSI MacBook keyboard I haven't found a single button that produced a character in combination with `ctrl+_`.

I'll stick to this solution for the time being, but I'm open to revisiting it if any other issues are found.